### DISCUSSION TITLE
Different Volume Settings For Podcasts/Audiobooks

### DISCUSSION THREAD

**Discussion by darthrater78** [advplyr/audiobookshelf-app#922] [Ideas]
I usually listen to audiobooks at a different rate than podcasts. It would be super handy if I could dedicate a speed to each type separately.

**Comment by Dr-Blank** [comment]
- https://github.com/advplyr/audiobookshelf/issues/1173
- https://github.com/advplyr/audiobookshelf/issues/1785

related open issues 
you can track progress there
